content/docs/Getting started/_index.md

```diff
@@ -6,3 +6,7 @@ date: 2019-02-22
 description: >
   Guides for system admins deploying OpenCue components and installing dependencies
 ---
+
```

Comment:
Looks like this PR includes changes from one of your other branches. I'm guessing that you may have created the `env_variables` branch not from `master`, but from another work branch?

You should clean up your branch to make sure it only includes changes relevant to this PR. You might want to create a new, clean branch from `master` and cherry-pick the relevant commits from `env_variables` to the new branch.

Reply:
yes @bcipriano, actually I had an open PR from master at the time I created this PR, thus this issue arise. 
I'll cherry pick the changes to a new branch and make a new PR accordingly.